### DISCUSSION TITLE
Problem with Speed managed on takeoff or descend ?

### DISCUSSION THREAD

**Discussion by DecMoonFR** [flybywiresim/aircraft#2348] [Help]
Hi, since a little time (but don't know when) I've got problem with A/THR

I have a recent dev build.
2020-11-12T02:54:20+00:00

On takeoff ;
- FLAPS 1 (so Vfe at 230 kts)

At 1000 feets and around 200kts I activat AT and engage A/THR by descent command to CL

1st : Flaps should be retract automatiquely at 210kts but it's not my problem
2nd : Plane continue to accelerate to 280 kts (but I'm with flaps and A/THR should never accelerate above Vfe, and bellow FL100, so speed should but at 250 kts max on managed, and it's the same if i'm on selected 220) After going to 280kts he begin to decelerate on good speed.

On approach :
I'm at FL80 at 180 with A/THR and speed selected to 180.
I put 3000ft on Altitude selected (not managed), and planez go down with high VS and accelerate, sometime above 250kts.

Any Idea ?

**Comment by uberbudd** [comment]
Hmm, I've never had my flaps retract automatically; there are indications in the PFD as to when to retract the flaps. Where are your throttle and speed settings set to managed (three dashes and circle)? I set my throttle at FLX/MCT for take off, then about 1000 ft above ground I switch to CL and turn the AP on. I generally leave my speed setting is set to managed. The FMC controls the flight from there with me only having to make adjustments to my altitude. I leave the throttle in CL until about 50-100 ft above touchdown.

**Comment by DecMoonFR** [comment]
Me neather, but it's in specs : Flaps Auto-retraction speed https://safetyfirst.airbus.com/control-your-speed-during-climb/ "
For example, an A320 at a Take-Off Weight (TOW) of 76T, S speed of 205 kts, the pilot will order flaps retraction most probably at or slightly above 210 kts, which is precisely the Flaps Auto-retraction speed"

However, i do the same. FLX/MCT for takeof, and about 1000ft engage AP. For CL it depend of the blinking CL. I've got it near 200kts. Then on CL, speed continue to rise. I didn't have any problems for my last two flights. The only difference was SID were straights after takeoff. But with a turn short after takeoff, I always have my speed problem.

**Reply by mga010** [reply]
Don't know if the Neo has a 1+F setting. The A320 has it and reduces to 1 automatically. But the slats are not retracted automatically.

I did not experience the problem with overspeeding below 10000 you report, nor the wrong speed at OP DES. Are you sure your FMA indicated the correct settings as you set them? However, I have the recent developer settings which could already be advanced from the one 6 days ago. This is a moving target.

**Reply by hotshotp** [reply]
ive always had my flaps retract - however like a pilot said it retracts at around 220 knots and not the slats, you go flaps zero yourself, thats how the real aircraft is.
